db/crud.go

```diff
@@ -1704,7 +1704,7 @@ func (db *DatabaseCollectionWithUser) SyncFnDryrun(ctx context.Context, body Bod
 	if docID != "" {
 		if docInBucket, err := db.GetDocument(ctx, docID, DocUnmarshalAll); err == nil {
 			oldDoc = docInBucket
-			if doc._body == nil {
+			if doc._body == nil || len(doc._body) == 0 {
```

Comment:
nit about go: checking for nil is redundant

```suggestion
			if len(doc._body) == 0 {
```

Reply:
done